packages/test/test-end-to-end-tests/src/test/mapEndToEndTests.spec.ts

```diff
@@ -375,6 +389,25 @@ describeNoCompat("SharedMap orderSequentially", (getTestObjectProvider) => {
         });
     });
 
+    itExpects.skip("Should close container when sending an op while processing another op",
```

Comment:
`skip`?

Reply:
Since the option in DeltaManager is defaulted to false, this error being expected won't be thrown.